core/App/components/misc/ConnectionNotification.tsx

```diff
@@ -0,0 +1,119 @@
+import { useNavigation } from '@react-navigation/core'
+import { StackNavigationProp } from '@react-navigation/stack'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Modal, StyleSheet, Text, View } from 'react-native'
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons'
+
+import { useTheme } from '../../contexts/theme'
+import { RootStackParams, Screens, Stacks } from '../../types/navigators'
+
+import InfoBox, { InfoBoxType } from './InfoBox'
+import UnorderedList from './UnorderedList'
+
+interface ConnectionNotificationProps {
+  connectionID?: string
+}
+
+const ConnectionNotification: React.FC<ConnectionNotificationProps> = ({ connectionID }) => {
+  const { t } = useTranslation()
+  const { ListItems, ColorPallet, TextTheme } = useTheme()
+  const [infoCardVisible, setInfoCardVisible] = useState(false)
+
+  const settingsNavigation = useNavigation<StackNavigationProp<RootStackParams>>()
+
+  const styles = StyleSheet.create({
+    modalCenter: {
+      flex: 1,
+      justifyContent: 'center',
+      alignItems: 'center',
+      backgroundColor: 'rgba(0,0,0,0.5)',
```

Comment:
We haven't consistently applied an opaque background to the modal dialogs that I know of

Reply:
I think the wire frames call for it here: https://xd.adobe.com/view/3db881d4-e963-42ec-a2c4-6b5ca23862a3-68eb/screen/a523d262-2549-4da9-9c2e-35ba1a84d7e8/ The background won't be opaque it will be translucent

Reply:
Was mostly referring to the opacity setting for the background colour. I think maybe just confirm with the UI/UX team if we want to have this as a one off or should we use this as an opportunity to apply the same effect across all full screen modal dialogs like this.

Reply:
@amanji Kim said this is the desired design for the modal

Reply:
Please add a translucent backdrop consistently across all modals of this type. Thank you!

Reply:
@knguyenBC by `this type` do you mean all connection notification modals? or all popup modals in general?

Reply:
@wadeking98 All popup modals in general

Reply:
Got it. Could I do that update in a separate PR? I feel it might be a bit out of scope for this PR

Reply:
That sounds like a good plan.